src/mono/mono/metadata/class-internals.h

```diff
@@ -73,10 +88,7 @@ struct _MonoMethod {
 	unsigned int is_generic:1; /* whenever this is a generic method definition */
 	unsigned int is_inflated:1; /* whether we're a MonoMethodInflated */
 	unsigned int skip_visibility:1; /* whenever to skip JIT visibility checks */
-	unsigned int verification_success:1; /* whether this method has been verified successfully.*/
-	unsigned int is_reabstracted:1; /* whenever this is a reabstraction of another interface */
-	unsigned int is_covariant_override_impl:1; /* whether this is an override with a signature different from its declared method */
-	signed int slot : 15;
+	signed int slot : 16;
 
```

Comment:
This should be : 18 then.

Reply:
Oddly verbose.
`unsigned int` can be written as just `unsigned` or `guint`.
`signed int` can be written `int` or `gint`.
There is no ambiguity or change of meaning.

Reply:
> This should be : 18 then.

I think I'd rather make a 2 bit `unused` field and leave `slot : 16`